api/resources/application.conf

```diff
@@ -0,0 +1,9 @@
+ktor {
+    deployment {
+        port = 8080
+        port = ${?PORT}
```

Comment:
what is this extra port doing here

Reply:
this file was generated by idea automatically